index.bs

```diff
@@ -2125,16 +2118,15 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
         :   If the user exercises a user agent user-interface option to cancel the process,
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
-            and [=set/remove=] |authenticator| from |issuedRequests|. Return a {{DOMException}} whose name is "{{NotAllowedError}}".
+            and [=set/remove=] |authenticator| from |issuedRequests|. Throw a "{{NotAllowedError}}" {{DOMException}}.
 
-        :   If the {{CredentialRequestOptions/signal}} member is present and the [=AbortSignal/aborted flag=] is set to
-            [TRUE],
+        :   If the {{CredentialRequestOptions/signal}} member is present and [=AbortSignal/aborted=],
```

Comment:
The first "the" seems a little out of place here. Let's also use the same phrasing as above.

```suggestion
        :   If <code>|options|.{{CredentialRequestOptions/signal}}</code> is present and [=AbortSignal/aborted=],
```